chalice/app.py

```diff
@@ -200,6 +189,15 @@ def maybe_fake_504() -> typing.Optional[chalice.Response]:
         res_headers.pop("Content-Length", None)
         res_headers["Strict-Transport-Security"] = "max-age=31536000; includeSubDomains; preload"
         res_headers["X-AWS-REQUEST-ID"] = app.lambda_context.aws_request_id
+
+        if status_code in (requests.codes.server_error,         # 500 status code
+                           requests.codes.bad_gateway,          # 502 status code
+                           requests.codes.service_unavailable,  # 503 status code
+                           requests.codes.gateway_timeout       # 504 status code
+                           ) \
+                and not (app.current_request.method == 'POST' and path.startswith('/v1/bundles')):
```

Comment:
can we define the checks at the top of the function?
```
def maybe_fakie_504:
  eligible_status_codes = (requests.codes.server_error,         # 500 status code
                           requests.codes.bad_gateway,          # 502 status code
                           requests.codes.service_unavailable,  # 503 status code
                           requests.codes.gateway_timeout )          # 504 status code
                          
  ignore_uri_types =  ('/v1/bundles')
  ignore_request_methods = ('POST')

# some time later
    if status code in eligible_status_codes: ... blah
```
not sure if this might be easier to read this way....
nit.

Reply:
Should be addressed.  Let me know what you think.